components/datetime/tests/patterns/tests/time_zones.json

```diff
@@ -882,5 +882,48 @@
         "expected": ["+05:45"]
       }
     ]
+  },
+  {
+    "locale": "bn",
+    "datetime": "2021-01-11T12:00:00.000+05:45[Asia/Kathmandu]",
+    "expectations": [
+      {
+        "patterns": [
+          "Z",
+          "ZZ",
+          "ZZZ"
+        ],
+        "configs": [
+          {
+            "Iso8601": [
+              "Basic",
+              "Required",
+              "Optional"
+            ]
+          }
+        ],
+        "expected": ["+0545"]
```

Comment:
Praise: Thanks for following and testing the spec.

Localized GMT:

> Note: The digits should be whatever are appropriate for the locale used to format the time zone, not necessarily from the western digits, 0..9. For example, they might be from ०..९.

ISO-8601 styles:

> Note: The digits in this case are always from the western digits, 0..9.